libobs/obs-properties.c

```diff
@@ -271,6 +271,30 @@ obs_property_t *obs_properties_get(obs_properties_t *props, const char *name)
 	return NULL;
 }
 
+void obs_properties_remove(obs_properties_t *props, const char *name)
+{
+	if (!props)
+		return;
+
+	// obs_properties_t is a forward-linked-list, so we need to keep both
```

Comment:
This should use the block comment syntax /* */

Reply:
Changed to match code style.